.coafile

```diff
@@ -27,6 +27,10 @@ bears = LineLengthBear
 [all.links]
 bears = InvalidLinkBear
 
+[all.unusedcode]
```

Comment:
No need to make a separate section. You can include it under `[all.python]`.
Also, read the [commit guidelines](https://api.coala.io/en/latest/Developers/Writing_Good_Commits.html) and modify your commit according to it.

Reply:
Okay I will do the changes.